ps_emailsubscription.php

```diff
@@ -1264,7 +1264,7 @@ public function hookActionDeleteGDPRCustomer($customer)
             if (Db::getInstance()->execute($sql)) {
                 return json_encode(true);
             }
-            return json_encode($this->l('Newsletter subscription: Unable to delete customer using email.'));
+            return json_encode($this->trans('Newsletter subscription: Unable to delete customer using email.', array(), 'Modules.Emailsubscription.Admin'));
```

Comment:
Can we take this opportunity to explicit the wording also? I don't understand if the deletion regards the customer or its email. Also, the uppercase is here useless after the colon. :-)

Reply:
Checked with Alex Darmon, please turn the current wording into _Newsletter subscription: no email to export, this customer has not registered._